Annotato/Annotato/ViewModels/Document/DocumentViewModel.swift

```diff
@@ -11,6 +11,7 @@ class DocumentViewModel: ObservableObject {
     private(set) var pdfDocument: PdfViewModel
 
     @Published private(set) var addedAnnotation: AnnotationViewModel?
+    @Published private(set) var deletedAnnotation: Annotation?
```

Comment:
Would it be better if the deletion of annotations be listened in the `AnnotationViewModel` instead? We can listen in AnnotationViewModel and if the deletedAnnotation id is equals to the view model's id, we can set the `isRemoved` flag in `AnnotationViewModel`. 

This way we don't have to handle the deletion in `DocumentView` and we can just make use of the subscriber in `AnnotationView`.